lightning/src/ln/chan_utils.rs

```diff
@@ -140,10 +140,10 @@ pub fn build_closing_transaction(to_holder_value_sat: u64, to_counterparty_value
 /// Implements the per-commitment secret storage scheme from
 /// [BOLT 3](https://github.com/lightningnetwork/lightning-rfc/blob/dcbf8583976df087c79c3ce0b535311212e6812d/03-transactions.md#efficient-per-commitment-secret-storage).
 ///
-/// Allows us to keep track of all of the revocation secrets of counterarties in just 50*32 bytes
+/// Allows us to keep track of all of the revocation secrets of counter parties in just 50*32 bytes
```

Comment:
FYI fixed a typo here

Reply:
`counterparty` is one word, not two, so I believe `counterparties` would be correct.

Reply:
changed, my spellchecker didn't like `counterparties` I guess I'll have to teach him :)